src/Factory/buildProxy.php

```diff
@@ -0,0 +1,24 @@
+<?php
+declare(strict_types=1);
+
+namespace Moka\Factory;
+
+use Moka\Exception\InvalidArgumentException;
+use Moka\Exception\MockNotCreatedException;
+use Moka\Generator\ProxyGenerator;
+use Moka\Proxy\ProxyInterface;
+use Moka\Strategy\MockingStrategyInterface;
+
+/**
+ * @param string $fqcn
+ * @param MockingStrategyInterface $mockingStrategy
+ * @return ProxyInterface
+ *
+ * @throws \ReflectionException
+ * @throws InvalidArgumentException
+ * @throws MockNotCreatedException
+ */
+function buildProxy(string $fqcn, MockingStrategyInterface $mockingStrategy): ProxyInterface
+{
+    return (new ProxyGenerator($mockingStrategy))->get($fqcn);
```

Comment:
Why aren't we caching proxy generators anymore?

Reply:
Because I've noticed that we have only _n_ proxy generator where _n_ is the number of strategies and probably the caching is useless for improving performance.

Reply:
What if we're creating multiple mock objects in a test? Without a cache we're also creating multiple generators.